flowey/flowey_core/src/pipeline.rs

```diff
@@ -552,6 +553,7 @@ impl Pipeline {
             cond_param_idx: None,
             ado_pool: None,
             ado_variables: BTreeMap::new(),
+            ado_job_name_overrides: BTreeMap::new(),
```

Comment:
this doesn't seem to be used anywhere (seems like a vestigial dupe from an iteration before you hoisted the map to the Pipeline level, where it should be?)